src/imagery/i.eodag/i.eodag.py

```diff
@@ -1323,13 +1323,6 @@ def main():
 
 
 if __name__ == "__main__":
```

Comment:
Maybe we can add this after the `try except` block to force the usage of EODAG 3:

```py
if int(eodag.__version__.split(".")[0]) < 3:
    gs.fatal(
        _(
            "eodag 3.0.0 or higher is required. eodag {} was found.".format(
                eodag.__version__
            )
        )
    )
```
Not sure if there is a different format for these type of error messages?

Reply:
@ninsbl what do you think?

Reply:
as of now, and because I teach tomorrow, I'll merge the removal of the warning only